prody/proteins/interactions.py

```diff
@@ -1988,12 +2079,21 @@ def calcStatisticsInteractions(data, **kwargs):
     for element in elements:
         if element not in stats:
             values = [t[1] for t in interactions_list if t[0] == element]
-            stats[element] = {
-                "stddev": np.round(np.std(values),6),
-                "mean": np.round(np.mean(values),6),
-                "weight": np.round(float(len(values))/len(data), 6),
-                "energy": get_energy([element.split('-')[0][:3], element.split('-')[1][:3]], energy_list_type)
-            }
+            
+            try:
+                stats[element] = {
+                    "stddev": np.round(np.std(values),6),
+                    "mean": np.round(np.mean(values),6),
+                    "weight": np.round(float(len(values))/len(data), 6),
+                    "energy": get_energy([element.split('-')[0][:3], element.split('-')[1][:3]], energy_list_type)
+                }
+            except:
+                LOGGER.warn('energy information is not available for ', element.split('-')[0][:3], element.split('-')[1][:3])
```

Comment:
I agree a warning is a good idea but only if the except is because of that. Otherwise it makes things even harder to figure out

Reply:
I am not sure whether something else might generate a problem here, and if the problem appears, then the user will see a problematic pair of residues.

Reply:
But only if they analyse the residues or the logs. I don’t know if it’s better to have something fail or make entries with different values that could get missed and/or make something else fail downstream

I guess there’s nothing we can do about it if we’re missing residue pairs. Maybe we can have something in the doc string about not having energies for all pairs with CS energies

Reply:
Let's wait for Carlos. It might be added. If not, I can add later an additional description.

Reply:
Ok, sounds good